libmscore/timesig.cpp

```diff
@@ -589,8 +589,10 @@ QString TimeSig::accessibleInfo()
       switch (timeSigType()) {
             case TimeSigType::FOUR_FOUR:
                   timeSigString = tr("Common time");
+                  [[gnu::fallthrough]];
             case TimeSigType::ALLA_BREVE:
                   timeSigString = tr("Cut time");
+                  [[gnu::fallthrough]];
             default:
```

Comment:
It definitly is a bug. In the Status line a common time time sig is shown as 4/4 and a cut time as 2/2. If that were intetional, the entire Switch would not be needed at all.

See #3195